src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -43,12 +45,43 @@
      */
     private static final String MAX = "max";
 
+    /**
+     * Map that mapping between a check and its ancestor token types.
+     */
+    private static final Map<String, List<Integer>> CHECK_TO_ANCESTOR_NODES_MAP =
+            new HashMap<String, List<Integer>>() {
+        {
+            put("ArrayTrailingComma", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("AvoidNestedBlocks", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("CommentsIndentation", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("DefaultComesLast", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("DeclarationOrder", Arrays.asList(TokenTypes.CLASS_DEF));
+            put("EmptyStatement", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("EqualsHashCode", Arrays.asList(TokenTypes.CLASS_DEF));
+            put("FallThrough", Arrays.asList(TokenTypes.METHOD_DEF));
+            put("FinalLocalVariable", Arrays.asList(TokenTypes.METHOD_DEF,
+                    TokenTypes.VARIABLE_DEF));
+            put("InnerTypeLast", Arrays.asList(TokenTypes.CLASS_DEF));
+            put("JavadocVariable", Arrays.asList(TokenTypes.CLASS_DEF));
+            put("MissingOverride", Arrays.asList(TokenTypes.CLASS_DEF));
+            put("MissingJavadocMethod", Arrays.asList(TokenTypes.CLASS_DEF));
```

Comment:
As discussed in https://github.com/checkstyle/patch-filters/issues/302#issuecomment-679387328, please remove these checks mapping to CLASS_DEF. Probably all CLAS_DEF mapping should be deleted, let's think

Reply:
@baratali please have a look https://github.com/checkstyle/patch-filters/issues/302#issuecomment-679514926